openapi/resource-server.yaml

```diff
@@ -1324,6 +1424,27 @@ components:
           kty: OKP
           crv: Ed25519
           x: oy0L_vTygNE4IogRyn_F5GmHXdqYVjIXkWs2jky7zsI
+    feeStructure:
+      title: feeStructure
+      type: object
+      properties:
+        fixed:
+          $ref: ./schemas.yaml#/components/schemas/amount
```

Comment:
Not directly related to the PR, but the description for the amount schema seems odd:

"All amounts are maxima, i.e. multiple payments can be created under a grant as long as the total amounts of these payments do not exceed the maximum amount per interval as specified in the grant."

Reply:
I guess we didn't update it properly when moving it to the shared schema. I'll fix that in this PR as well.